ci/kokoro/macos/builds/cmake-vcpkg.sh

```diff
@@ -71,9 +71,17 @@ io::log_h2 "Compiling with ${NCPU} cpus"
 cmake --build "${BINARY_DIR}" -- -j "${NCPU}"
 
 io::log_h2 "Running unit tests"
+ctest_args=(
+  # Print the full output on failures
+  --output-on-failure
+  # Run many tests in parallel, use -j for compatibility with old versions
+  -j "$(nproc)"
+  # Make the output shorter on interactive tests
+  --progress
+)
 (
   cd "${BINARY_DIR}"
-  ctest -LE integration-test --output-on-failure -j "${NCPU}"
+  ctest "${ctest_args[@]}" -LE "integration-test"
```

Comment:
Can we use env -C "${BINARY_DIR}" ctest ..., instead of the sub-shell, to match the other files?

Reply:
Same thing, macOS does not support this flag (just tried on my laptop).